modules/passkey/package.json

```diff
@@ -48,24 +48,24 @@
   "devDependencies": {
     "@noble/curves": "^1.4.0",
     "@nomicfoundation/hardhat-ethers": "^3.0.6",
-    "@nomicfoundation/hardhat-network-helpers": "^1.0.10",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.11",
     "@nomicfoundation/hardhat-toolbox": "^5.0.0",
     "@safe-global/mock-contract": "^4.1.0",
     "@safe-global/safe-4337": "workspace:^0.3.0-1",
     "@safe-global/safe-4337-local-bundler": "workspace:^0.0.0",
     "@simplewebauthn/server": "^10.0.0",
-    "@types/node": "^20.14.0",
+    "@types/node": "^20.14.8",
     "dotenv": "^16.4.5",
-    "ethers": "^6.12.1",
-    "hardhat": "^2.22.3",
+    "ethers": "^6.13.1",
+    "hardhat": "^2.22.5",
     "hardhat-deploy": "^0.12.4",
     "solhint": "^5.0.1",
     "ts-node": "^10.9.2",
-    "typescript": "^5.4.5"
+    "typescript": "^5.5.2"
   },
   "dependencies": {
-    "@account-abstraction/contracts": "^0.7.0",
-    "@openzeppelin/contracts": "^5.0.0",
+    "@account-abstraction/contracts": "0.7.0",
+    "@openzeppelin/contracts": "^5.0.2",
     "@safe-global/safe-contracts": "^1.4.1-build.0",
```

Comment:
I think this should also have strict dependencies.

Reply:
From what I found, passkeys only used safe contracts for testing and a version update wouldn't result in a new bytecode, so I reckon it's fine as is. I can pin it though

Reply:
In that case, shouldn't these be `devDependencies`?